index.html

```diff
@@ -1085,6 +924,213 @@ <h3>Implicit Release of Pointer Capture</h3>
+            the user agent and the prediction algorithm it uses.</p>
+                
+            <p>The events in the predicted event list will have increasing
+            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>([[!WHATWG-DOM]]),
+            so the first event will have the smallest <code>timeStamp</code> which should still be greater than
+            the last event in the <a>coalesced event list</a>.</p>
+
+            <pre id="example_11" class="example" title="Conceptual approach to drawing using coalesced events and predicted events">
+                <code>
+                var predicted_points = [];
+                window.addEventListener("pointermove", function(event) {
+                    // Clear the previously drawn predicted points.
+                    for (let e of predicted_points.reverse()) {
+                        clearPoint(e.pageX, e.pageY);
+                    }
+
+                    // Draw the actual movements that happened since the last received event.
+                    for (let e of event.getCoalescedEvents()) {
+                        drawPoint(e.pageX, e.pageY);
+                    }
+                
+                    // Draw the current predicted points to reduce the perception of latency.
+                    predicted_points = event.getPredictedEvents();
+                    for (let e of predicted_points) {
+                        drawPoint(e.pageX, e.pageY);
+                    }
+                });
+                </code>
+            </pre>
+        </section>
+        
+        <section>
+            <h2><dfn>Populating and maintaining the coalesced and predicted event lists</dfn></h2>
+            <p>When a <a>PointerEvent</a> is created, run the following steps for each event in the
+            <a>coalesced event list</a> and <a>predicted event list</a>:</p>
+            <ol>
+                <li>Set the event's <code>pointerId</code>, <code> pointerType</code>,
+                <code>isPrimary</code> and <code>isTrusted</code> to the <var>PointerEvent</var>'s
+                <code>pointerId</code>, <code> pointerType</code>, <code>isPrimary</code> and
+                <code>isTrusted</code>.</li>
+                <li>Set the event's <code>cancelable</code> and <code>bubbles</code> attributes to false.</li>
+                <li>Set the event's <a>coalesced event list</a> and <a>predicted event list</a> to an empty list.</li>
+                <li>Initialize the rest of the attributes the same way as <a>PointerEvent</a>.</li>
+            </ol>
+               
+            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events targets dirty</dfn> and an associated
+            <dfn>predicted events targets dirty</dfn> flag. When an event is created they must be initialized to false.</p>
+                
+            <p>When the <a>PointerEvent</a>'s target is changed, set <a><code>coalesced events targets dirty</code></a>
```

Comment:
Agree, this would simplify things here as well, bascially just boiling down to the statement that the target of coalesced and predicted events is always the same as its "parent" event, and that if the coalesced or predicted event was over another target, to ignore this but to set the relevant dirty flag. I'll think of a clean way to word this...

Reply:
> is it possible for a developer to observe when the target of these events change?

it further makes me think: is the dirty flag only set when the `getCoalescedEvents` / `getPredictedEvents` methods get called (which is what the current text suggests), or is this happening as the original / high-level / coalesced pointer event gets fired?

additionally, i've never been quite clear...where do i "find" or access this flag? is this something internal to the browser itself and not meant to be actually exposed to JS authors? this concept of the flag seems rather vague to me...

Reply:
(incidentally, we can either tackle this here in this PR, or make a separate issue/PR for this aspect ... whatever folks think is cleaner)

Reply:
Yes, as far as I can tell the flag is an internal implementation optimization, which is why I suggest we remove it from the spec and just say that we always copy the current target to the coalesced / predicted points.

I'm okay with leaving this for a new PR since it's orthogonal to the changes being made here.

Reply:
so purely from a JS author's point of view, what matters really is, in essence: all events in the coalesced event list and the predicted event list MUST have the same target as their "parent" pointer event (and the rest is left up to implementers on how that is achieved). maybe the whole flag business can be shuffled into a separate sub-section aimed at implementers here? (just to make the distinction clearer - at least mentioning the existence of the flag, but not going into any more detail)

Reply:
would also be good to come up with some name for what i call "parent" pointer event above. not sure what the generally accepted naming for this sort of thing is...

Reply:
@mustaqahmed @smaug---- any thoughts on these aspects? should we tackle them in a separate PR?

Reply:
we'll tackle this separate from https://github.com/w3c/pointerevents/issues/370